test/integration/test-visibility-states.js

```diff
@@ -124,18 +127,28 @@ t.run('Viewer Visibility State', () => {
             img.setAttribute('width', 100);
             img.setAttribute('height', 100);
             img.setAttribute('layout', 'fixed');
-            win.document.body.appendChild(img);
+            if (!V1_IMG_DEFERRED_BUILD) {
```

Comment:
Let's add TODOs to remove these branches in the file when the experiment is done?